dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf

```diff
@@ -20,10 +20,17 @@ streamingModelConfig {
   }
 
   pmml {
+    fileExtension: ".pmml"
     modelsDirectory: "http://pmml-proxy:5100"
     modelDirectoryHrefSelector: "body > ul > li > a"
   }
 
+  h2o {
+    fileExtension: ".mojo"
+    modelsDirectory: "http://lol"
```

Comment:
xd is this still WiP PR?

Reply:
we dont have any h2o repository, so it's some dummy path :p

Reply:
we do have local file repository, it's in exports folder in h2o dev-environment